pkg/datasource/opensergo/demo/datasource_opensergo_example.go

```diff
@@ -0,0 +1,121 @@
+// Copyright 1999-2020 Alibaba Group Holding Ltd.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package main
+
+import (
+	"fmt"
+	sentinel "github.com/alibaba/sentinel-golang/api"
+	"github.com/alibaba/sentinel-golang/core/base"
+	"github.com/alibaba/sentinel-golang/core/config"
+	_ "github.com/alibaba/sentinel-golang/ext/datasource"
+	"github.com/alibaba/sentinel-golang/logging"
+	"github.com/alibaba/sentinel-golang/pkg/datasource/opensergo"
+	_ "github.com/alibaba/sentinel-golang/pkg/datasource/opensergo"
+	"github.com/alibaba/sentinel-golang/util"
+	"log"
+	"math/rand"
+	"sync/atomic"
+	"time"
+)
+
+const (
+	host string = "33.1.33.1"
+	port int    = 10246
+
+	namespace string = "default"
+	app       string = "foo-app"
+)
+
+type Counter struct {
+	pass  *int64
+	block *int64
+	total *int64
+}
+
+func main() {
+
+	openSergoDataSource, _ := opensergo.NewOpenSergoDataSource(host, port, namespace, app)
+	openSergoDataSource.Initialize()
+	openSergoDataSource.Start()
```

Comment:
start func and RegisterSubscribe funcs can be in Initialize func

Reply:
ok. I will merge them, and remove the start func

Reply:
done.